### ISSUE TITLE
Feature request: reader plugin for zarr files output by empanada-napari

### ISSUE THREAD

**Issue by GenevieveBuckley** [volume-em/empanada-napari#21]
It would be helpful if the empanada-napari plugin provided an inbuilt way to read back in prediction data generated by itself. The standard zarr reader plugins work in some, but not all cases for data generated by empanada-napari.

If there is just one dataset group in the zarr file, the regular zarr reader plugins can typically handle that. However, in cases where there are multiple datasets being saved by empanada-napari into a single zarr file (for example, when each orthoslice prediction output is saved individually, alongside the final combined predictions), that doesn't work.

Typically, those plugins find multiple datasets and attempt to parse it as a multi-resolution dataset, and fail because the data shape is the same for all of them (instead of steadily decreasing in size, like a multiscale zarr array).

I often want to re-load previous results from empanada-napari, in order to use the split and merge tools at a later date. Not being able to always read the data back in easily makes this workflow clunkier than is ideal.

**Comment by conradry** [comment]
I agree that reloading data isn't as easy as it should be. Data exporting also needs improvements, especially for saving metadata.

I really don't want to complicate things by creating new readers and writers though. I'm leaning towards outputting the results in OME-Zarr format. The reader plugin for that [already exists](https://github.com/ome/napari-ome-zarr) and works well. Another advantage of OME-Zarr is that I can store detailed label metadata which will allow all the proofreading tools (except paint and erase) to work without loading the entire segmentation into memory.  I'll do some experiments and try to incorporate that into the next release.

**Comment by GenevieveBuckley** [comment]
> I really don't want to complicate things by creating new readers and writers though. I'm leaning towards outputting the results in OME-Zarr format. The reader plugin for that [already exists](https://github.com/ome/napari-ome-zarr) and works well.

I agree with you actually. It does seem better to leave the reader plugins to handle reading data, and make the output format here something they can understand.

Mostly I wasn't sure if you have some special reason for needing things arranged differently in the empanada output, hence this suggestion.
